bindings/python/python/opendal/__init__.pyi

```diff
@@ -282,19 +284,31 @@ class AsyncOperator(_Base):
         Returns:
             True if the object exists, False otherwise.
         """
-    async def list(self, path: PathBuf) -> AsyncIterable[Entry]:
+    async def list(
+        self, path: PathBuf, *, start_after: str | None = None
+    ) -> AsyncIterable[Entry]:
         """List the objects at the given path.
 
         Args:
             path (str|Path): The path to the directory.
+            start_after (str | None): The key to start listing from.
 
         Returns:
             An iterable of entries representing the objects in the directory.
         """
-    async def scan(self, path: PathBuf) -> AsyncIterable[Entry]: ...
-    async def presign_stat(
-        self, path: PathBuf, expire_second: int
-    ) -> PresignedRequest:
+    async def scan(
```

Comment:
Scan should be replaced by `list(recursive=True)`. We can add this for list first.

Reply:
So anything to do on this pr?